db/migrate/20160620205641_add_property_attributes_to_generic_object.rb

```diff
@@ -0,0 +1,5 @@
+class AddPropertyAttributesToGenericObject < ActiveRecord::Migration[5.0]
+  def change
+    add_column :generic_objects, :property_attributes, :jsonb, :default => {}
```

Comment:
@Fryguy  It is not required. But the default makes it easier to set/get the attribute without check for nil.